cli/tests/unit/os_test.ts

```diff
@@ -188,3 +188,9 @@ unitTest({ perms: { env: true } }, function systemMemoryInfo(): void {
   assert(info.swapTotal >= 0);
   assert(info.swapFree >= 0);
 });
+
+unitTest({ perms: { env: true } }, function systemCpuInfo(): void {
+  const { cores, speed } = Deno.systemCpuInfo();
+  assert(cores >= 0);
+  assert(speed >= 0);
```

Comment:
Are there situation where this API might return `0` for any of the fields? If so it should be mentioned in jsdoc for the API

Reply:
I don't think so but I guess technically the speed could be < 1MHz. Cores should always be more than 0 so I will change that. (looked at the rust tests for `sys_info` and they seem test if its more than 0 for both)